File: Python/total_words.py
#!/usr/bin/python3

import sys
import re

nb_of_words = 0

for line in sys.stdin:
    lst = re.findall(r'[a-zA-Z]+', line)
    nb_of_words += len(lst)

print('{} words'.format(nb_of_words))

File: Perl/total_words.pl
#!/usr/bin/perl -w

chomp(@lines = <STDIN>);


$nb_of_words = 0;
foreach $line(@lines){
	@lst = split /[^a-zA-Z]/, $line;

	foreach $var (@lst){
		if($var =~ m/^[a-zA-Z]+$/){
			$nb_of_words++;
		}
	}
}
print "$nb_of_words words\n";

